keps/sig-docs/20191020-doc-policies-for-third-party-content.md

```diff
@@ -0,0 +1,339 @@
+
+1. Is there a list of content pages that are so focused on third party product
+usage that they should be removed or updated from the Kubernetes documentation?
+
+See https://github.com/kubernetes/website/issues/15748.
+
+1. When should the Kubernetes documentation host third party content that isn't
+maintained by a Kubernetes SIG?
+
+As infrequently as possible, with linking preferred to hosting. 
+
+1. How does the Kubernetes project handle third party content that is not kept
+up to date or hosts?
+
+If content isn't refreshed within 180 days, notify stakeholders of 90 days to 
+update content or migrate it elsewhere before removing it.
+
+1. Can feature owners flag when third party content is *required*, as opposed to
+preferable or common?
+
+Is this capability required for KEP approval?
+
+1. Who decides when to include third-party content?
+
+SIGs responsible for particular features can include third-party content at
+their discretion, preferably by linking to the third party's own documentation.
+
+1. What standard of quality and review must be met before docs include
+third-party content?
+
+Third-party content must be necessary for Kubernetes to function in-project.
+
+1. To what extent should SIG Docs advocate for third-party content providers to 
+host their own content, or decline to host third-party content altogether?
+
+Kubernetes docs publish third-party content only if:
+
+- It's necessary for Kubernetes to function. For example: container runtimes 
+(containerd, CRI-o, Docker), networking policy (CNI plugins), Ingress 
+controllers, and logging.
+
+- It's an applied example of another project in the Kubernetes GitHub org. This
+includes the [kubernetes](https://github.com/kubernetes) and 
+[kubernetes-sigs](https://github.com/kubernetes-sigs) repositories.
+
+Third-party content should be linked instead of hosted whenever possible. 
+
+</del>
+
+1. Clearly define what documentation is required so that readers understand
+   how to deploy, operate and consume Kubernetes clusters using features from
+   in-project code and its mandatory dependencies.
+
+### Non-Goals
+
+1. Outright removal of all content relating to vendors and projects
+   outside the CNCF ecosystem.
+
+## Proposal
+
+1. Revise the [content guide](https://github.com/kubernetes/website/blob/master/content/en/docs/contribute/style/content-guide.md#contributing-content) to achieve the KEP goal:
+
+- Specify that Kubernetes docs are limited to content required for Kubernetes to
+function in-project. Docs may include third-party content for components that 
+require a third-party solution to function. Docs may include content for 
+other projects in the Kubernetes org. Third-party content must be linked 
+whenever possible, rather than duplicated or hosted in k/website.
+
+2. Revise the documentation when the KEP is approved:
+
+- **Third-party content:** Notify stakeholders via GitHub issues in k/website
```

Comment:
Can you also please bunch up these removals and post to kubernetes-sig-docs mailing list as well? (since github notifications are busted for a bunch of folks)?